src/codec_svt.c

```diff
@@ -161,6 +161,17 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
             svt_config->profile = HIGH_PROFILE;
         }
 
+        // In order for SVT-AV1 to force keyframes by setting pic_type to
+        // EB_AV1_KEY_PICTURE on any frame, force_key_frames has to be set.
+        svt_config->force_key_frames = 1;
```

Comment:
Nit/optional: 1 => TRUE

Here is the relevant code in SVT-AV1:

```
    /**
     * @brief Signal that force-key-frames is enabled.
     *
     */
    Bool force_key_frames;
```

and

```
typedef uint8_t Bool;
#define FALSE 0
#define TRUE 1
```

Reply:
Done.